### ISSUE TITLE
Debugger getting unknown command

### ISSUE THREAD

**Issue by ZimboPro** [microsoft/vscode-cmake-tools#525]
### Brief Issue Summary

If I right click on executable in CMake Tools Pane and select _Run with debugger_, it builds but then doesn't run the debugger as in gives _lldb: unrecognized option `--interpreter=mi'_

### CMake Tools Log
[build] Build finished with exit code 0
[proc] Executing command: /usr/bin/lldb --version
[proc] Executing command: /usr/bin/lldb --version

### Platform and Versions

- **Operating System**:  macOS
- **CMake Version**:  3.11.3
- **VSCode Version**: 1.25.1
- **CMake Tools Extension Version**: 1.1.1
- **Compiler/Toolchain**: clang
- **LLDB Version**: lldb-900.0.64
                                 Swift-4.0

**Comment by madig** [comment]
I have the following launch.json configuration
```JSON
{
    "version": "0.2.0",
    "configurations": [

        {
            "name": "(lldb) Launch",
            "type": "cppdbg",
            "request": "launch",
            "program": "${workspaceFolder}/cxx/a.exe",
            "args": [
                "..."
            ],
            "stopAtEntry": false,
            "cwd": "${workspaceFolder}/cxx",
            "environment": [],
            "externalConsole": false,
            "MIMode": "lldb",
            "miDebuggerPath": "C:\\Program Files\\LLVM\\bin\\lldb.exe",
        }
    ]
}
```
And get the same error on Windows (no CMake, just F5ing).

**Comment by vector-of-bool** [comment]
@madig I'd take that one to the cpptools repo. They are the maintainers of the `cppdbg` debugging engine.

**Comment by bobbrow** [comment]
This was marked as fixed in 1.1.3.  If you continue to have issues with this, please let us know.

**Comment by jharrang** [comment]
I'm hitting this while running 1.1.3 with the following launch.json, using both gdb and lldb for MIMode:

"""
{
    "version": "0.2.0",
    "configurations": [
    {
        "name": "(lldb) Launch",
        "type": "cppdbg",
        "request": "launch",
        "program": "${command:cmake.launchTargetPath}",
        "args": [],
        "stopAtEntry": false,
        "cwd": "${workspaceFolder}",
        "environment": [],
        "externalConsole": false,
        "MIMode": "gdb",
        "visualizerFile": "${workspaceFolder}/eigen.natvis",
        "showDisplayString": true
    }]
}

**Comment by bobbrow** [comment]
The fix is not in 1.1.3, but should be in 1.2.0 which will be released next week.
